app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventDetailsActivity.java

```diff
@@ -0,0 +1,59 @@
+package de.tum.in.tumcampusapp.component.ui.ticket;
+
+import android.os.Bundle;
+import android.support.v4.view.ViewPager;
+
+import java.util.List;
+
+import de.tum.in.tumcampusapp.R;
+import de.tum.in.tumcampusapp.component.other.generic.activity.BaseActivity;
+import de.tum.in.tumcampusapp.component.ui.ticket.model.Event;
+import io.reactivex.disposables.CompositeDisposable;
+
+/**
+ * TODO: combine this with KinoActivity
+ */
+public class EventDetailsActivity extends BaseActivity{
+
+    public EventDetailsActivity() {
+        super(R.layout.activity_kino);
+    }
+
+    private final CompositeDisposable disposable = new CompositeDisposable();
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        // set up ViewPager and adapter
+        ViewPager mPager = findViewById(R.id.pager);
+
+        // TODO: extract key to const class
+        int clickedEventId = getIntent().getIntExtra("event_id", 0);
```

Comment:
The event ID you get here is wrong if in the Events List only booked events are shown. E.g. clicking on Unity3 will result in getting the ID for Unity2.
I guess that this problem has to be solved in the Event List or Event Card.

Reply:
@saileru @schulzalexander I just solved this problem last night.  And I commited it . The problem results from that the event_id is not corresponding to its name.(for example id = 0 with "Unity 2" in getbookedevents()